src/objects/Relationship__c.object

```diff
@@ -331,11 +331,19 @@ TEXT(Contact__r.Salutation) + &quot; &quot; + Contact__r.FirstName + &quot; &quo
     <searchLayouts/>
     <sharingModel>ReadWrite</sharingModel>
     <validationRules>
-        <fullName>Related_Contact_Do_Not_Change</fullName>
-        <active>true</active>
-        <description>Do not allow user to change Related Contact value as it damages the paired relationship data</description>
-        <errorConditionFormula>and(not( ISNEW()), ISCHANGED( RelatedContact__c ))</errorConditionFormula>
-        <errorDisplayField>RelatedContact__c</errorDisplayField>
-        <errorMessage>Rather than change the contacts in an existing relationship, delete this relationship and create a new on between the new contacts</errorMessage>
+            <fullName>Contact_Do_Not_Change</fullName>
+            <active>true</active>
+            <description>Do not allow users to change the Contact value as it damages the paired relationship data.</description>
+            <errorConditionFormula>AND(NOT( ISNEW() ), ISCHANGED(  Contact__c ))</errorConditionFormula>
+            <errorDisplayField>Contact__c</errorDisplayField>
+            <errorMessage>Rather than change the Contacts in an existing Relationship, delete this Relationship and create a new one.</errorMessage>
+    </validationRules>
+    <validationRules>
```

Comment:
Just a heads up... because we're making typo/text changes here in the other open PR, whichever PR is merged first will likely cause conflicts in the other one :\